ts/src/core/parser/index.ts

```
@@ -3,43 +1,138 @@
+          const files = await readdir(dirPath);
+          
+          for (const file of files) {
+            if (file.endsWith('.jsonl')) {
+              const filePath = join(dirPath, file);
+              const events = await parseJSONLFile(filePath, startTime, endTime);
+              allEvents.push(...events);
+            }
+          }
+        } catch (error) {
+          // Skip inaccessible directories
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)